stack.yaml

```diff
@@ -9,7 +9,9 @@ packages:
   - example
 flags: {}
 extra-deps:
+  - aeson-1.5.2.0
```

Comment:
why is this extra-depped in rather than using the lts-16.4 version?

Reply:
Whoops this was just me testing. Thanks for the feedback I'll get all these in later.